packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -124,6 +124,10 @@ final Animatable<double> _kScaleTween = Tween<double>(begin: 1.0, end: 1.0 - _kS
 /// means the sheet takes up only the bottom 10% of the screen. If not provided, defaults
 /// to 0.08 (8% of screen height).
 ///
+/// When `showDragHandle` is set to `true`, then an iOS-style drag handle will
```

Comment:
Isn't everything in this widget in iOS style?
```suggestion
/// When `showDragHandle` is set to `true`, then a drag handle will
```